src/msg.rs

```diff
@@ -213,6 +213,11 @@ pub enum HandleMsg {
         level: ContractStatusLevel,
         padding: Option<String>,
     },
+
+    // Permit
+    RevokePermit {
+        permit_name: String,
+    },
 }
```

Comment:
we should also have an "unrevoke" function imo

Reply:
Does it make more sense to "unrevoke" or to simply create a new permit?

Reply:
I don't like "unrevoke". It's not intuitive, and will only lead to confusion on how to use and what it means

Reply:
@Cashmaney I used "unrevoke" as a placeholder name, hence the quote marks, it wasn't the point. We can use "AllowPermit" for example, whatever works. The point is if a user accidentally blocks a permit, there should be a way to unblock it.

@dylanschultzie this ties in to your question. The thinking at the moment is that the `permit_name` will just be a constant value used by the service, for example its URL or just a name, like "secretswap.io" or "Secret Swap". I don't think websites should have a bunch of logic letting users to generate new permit names, but at most a simple "the permit to this site has been revoked. To continue using it please whitelist our service again [by clicking here]"

Reply:
I disagree. I think if a user blocks a permit, they can just create a new one. Re-allowing a permit complicates the logic and the UX. Can you elaborate on what advantage re-allowing the permit has vs. just creating a new one?

Reply:
@Cashmaney I'm thinking about secretswap, or the bridge for example. They'll probably use a constant when prompting the user to sign a permit, like "secret-swap" or "sercret-bridge" or whatever (as i explained above). If a user blocks one of them and goes back to the site, that will require us to add code to those sites to use a different string instead of the default one. It's easier for us to just ask for the permission to be returned, instead of the system getting messier.

Reply:
The permit is tied to the application name, rather than a UUID or something, then?

In my mind the process/implementation would've been something like:
```
permit: {
"name": string
"permit_id": string
"permissions": { ... }
}
```

where the name is independent of the binding. If the permission is revoked, that single `permit_id` is revoked and getting new permission would create a new `permit_id`. 

To be clear, I'm not pressing in favor of it, that was simply my thought.

Reply:
"unrevoke" is unnecessary IMO, "permissions" is interesting.